client/components/ShareModal.jsx

```diff
@@ -0,0 +1,50 @@
+import React from 'react';
+//import { } from './Styles.jsx';
+import styled from 'styled-components';
+
+const ModalWrapper = styled.div`
```

Comment:
May want to put this in your styled file. I prefer to keep my style classes separated from other modules, but that's up to you